### PR TITLE
Sound toggle

### PR DESCRIPTION
**Pull request by Verzijl**
Basis to be used in the settings menu in the GUI.

<feature

### CODE REVIEW

**Comment by Verzijl** [comment]
Fixes #255.

File: src/main/java/utility/Settings.java [hunk 1]
```diff
@@ -31,6 +31,8 @@
     public static final double JUMP_SPEED_WALKER = 3 * MONSTER_SPEED;
     public static final double JUMP_HEIGHT_WALKER = 200;
 
+    public static final boolean PLAY_SOUND = true;
```
Comment (toinehartman) [inline src/main/java/utility/Settings.java]: Should not be final, cannot be modified now.

< bug

**Comment by toinehartman** [comment]
:shipit:

**Comment by jimhommes** [comment]
If comments are fixed, :shipit:

<feature

**Pull request merged**